exercises/bowling/canonical-data.json

```diff
@@ -19,255 +19,362 @@
-    "property": "score",
-    "input": {
-      "previousRolls": [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 10, 7, 1]
+    {
+      "uuid": "e483e8b6-cb4b-4959-b310-e3982030d766",
+      "description": "a strike in the last frame gets a two roll bonus that is counted once",
+      "property": "score",
+      "input": {
+        "previousRolls": [
+          0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 10, 7, 1
+        ]
+      },
+      "expected": 18
     },
-    "expected": 18
-  }, {
-    "uuid": "9d5c87db-84bc-4e01-8e95-53350c8af1f8",
-    "description": "rolling a spare with the two roll bonus does not get a bonus roll",
-    "property": "score",
-    "input": {
-      "previousRolls": [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 10, 7, 3]
+    {
+      "uuid": "9d5c87db-84bc-4e01-8e95-53350c8af1f8",
+      "description": "rolling a spare with the two roll bonus does not get a bonus roll",
+      "property": "score",
+      "input": {
+        "previousRolls": [
+          0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 10, 7, 3
+        ]
+      },
+      "expected": 20
     },
-    "expected": 20
-  }, {
-    "uuid": "576faac1-7cff-4029-ad72-c16bcada79b5",
-    "description": "strikes with the two roll bonus do not get bonus rolls",
-    "property": "score",
-    "input": {
-      "previousRolls": [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 10, 10, 10]
+    {
+      "uuid": "576faac1-7cff-4029-ad72-c16bcada79b5",
+      "description": "strikes with the two roll bonus do not get bonus rolls",
+      "property": "score",
+      "input": {
+        "previousRolls": [
+          0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 10, 10, 10
+        ]
+      },
+      "expected": 30
     },
-    "expected": 30
-  }, {
-    "uuid": "72e24404-b6c6-46af-b188-875514c0377b",
-    "description": "a strike with the one roll bonus after a spare in the last frame does not get a bonus",
-    "property": "score",
-    "input": {
-      "previousRolls": [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 7, 3, 10]
+    {
+      "uuid": "72e24404-b6c6-46af-b188-875514c0377b",
+      "description": "a strike with the one roll bonus after a spare in the last frame does not get a bonus",
+      "property": "score",
+      "input": {
+        "previousRolls": [
+          0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 7, 3, 10
+        ]
+      },
+      "expected": 20
     },
-    "expected": 20
-  }, {
-    "uuid": "62ee4c72-8ee8-4250-b794-234f1fec17b1",
-    "description": "all strikes is a perfect game",
-    "property": "score",
-    "input": {
-      "previousRolls": [10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10]
+    {
+      "uuid": "62ee4c72-8ee8-4250-b794-234f1fec17b1",
+      "description": "all strikes is a perfect game",
+      "property": "score",
+      "input": {
+        "previousRolls": [10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10]
```

Comment:
in `bowling`, is there some rule I can use to understand when the elements of `previousRolls` are (or should be) on the same line as the `[` and `]` versus when they are on different lines?

Reply:
There isn't. I've made them consistent.